### PR TITLE
Intro mode

### PR DESCRIPTION
**Pull request by ivan-BRHM**
#14
Finished the intro mode/intro screen. Made graphics for them as well as interaction with them to start the game - that being all features needed. Code might need some refactoring...

### CODE REVIEW

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Looks good. Only a few minor changes needed.

File: my_game.py [hunk 1]
```diff
@@ -289,6 +297,16 @@ def destroy(self):
...
+    """
+    the state of the game: INTRO, IN_GAME or GAME_OVER
+    """
+
+    INTRO = 0
+    IN_GAME = 1
+    GAME_OVER = 2
+
+
 class MyGame(arcade.Window):
```
[the remark this answers is not in the document]
Comment (ivan-BRHM) [reply my_game.py:310]: you mean like this?
`class GameState(Enum):
    """
    the state of the game: INTRO, IN_GAME or GAME_OVER
    """

    INTRO
    IN_GAME
    GAME_OVER`

[the remark this answers is not in the document]
Comment (tobiasbp) [reply my_game.py:310]: Hmmm.... Actually I mean:
```
from enum import Enum, auto

INTRO = auto()
...
...

```

[the remark this answers is not in the document]
Comment (ivan-BRHM) [reply my_game.py:310]: like this?

`    INTRO = auto()
    IN_GAME = auto()
    GAME_OVER = auto()`

[the remark this answers is not in the document]
Comment (tobiasbp) [reply my_game.py:310]: yes

File: my_game.py [hunk 2]
```diff
@@ -392,89 +419,122 @@ def setup(self):
...
+                10,  # X position
+                SCREEN_HEIGHT - 20,  # Y_position
+                arcade.color.WHITE  # Color of text
+            )
+            arcade.draw_text(
+                "LIVES: {}".format(self.player_sprite.lives),  # Text to show
+                10,  # X position
+                SCREEN_HEIGHT - 45, # Y positon
+                arcade.color.WHITE  # Color of text
+            )
```
Comment (tobiasbp) [inline my_game.py:480]: Maybe you could put in a placeholder here for state _GAME_OVER_? Just put _pass_ in the _elif_ block.

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Looks great. One minor thing left.

**Review by tobiasbp — APPROVED** [review]

**Pull request merged**